### DISCUSSION TITLE
Sync Unchanged Records

### DISCUSSION THREAD

**Discussion by GhayasAbouFakher** [Azure/azure-mobile-apps#705] [Q&A] [answered]
Hello, I noticed that when I asked to sync tables, some of unchanged records synced whatever the version or updateAt was.
![image](https://github.com/Azure/azure-mobile-apps/assets/88130432/479483f6-1b5c-4fe9-a9cc-b92a7b448446)

this attached image shows one record has no changes since 2021 but it always exist in the synced items list.
So is there any reason for this behavior?
Regards!

**Comment by adrianhall** [comment, marked as answer]
Without some additional information about the way things are set up, I can't say.

The search says "find records where updatedAt > 2021-05-19T08:46:40.547Z"
The record holds the exact same date.  Since it's gt and not ge, it should not be included.

However, some databases don't have a resolution for comparison that fine.  Your database must be configured such that the comparison is on ms accuracy.  Essentially, the database gets queried for this comparison and returns the record.  The service just interprets this and returns it.

**Reply by GhayasAbouFakher** [reply]
I really appreciate.
we should use more than 3 numbers in ms part or adding ns.
Thank you anyway
